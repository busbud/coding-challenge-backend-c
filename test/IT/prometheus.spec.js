const request = require('supertest');

const App = require('../../app');

const { actuatorListener } = new App();

describe('GET /actuator/prometheus', () => {
    let response;

    beforeAll(async () => {
        response = await request(actuatorListener)
            .get('/actuator/prometheus')
            .send();
    });

    it('returns a 200', () => {
        expect(response.statusCode).toEqual(200);
    });

    it('returns prometheus metrics in body', () => {
        expect(response.body).toMatchSnapshot();
    });
});
