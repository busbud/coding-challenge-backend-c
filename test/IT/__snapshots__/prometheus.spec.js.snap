// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GET /actuator/prometheus returns prometheus metrics in body 1`] = `Object {}`;
